packages/alea-frontend/pages/api/get-blog-posts.ts

```diff
@@ -0,0 +1,16 @@
+import { Blog } from '@stex-react/api';
+import { executeDontEndSet500OnError } from './comment-utils';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const blogs: Blog[] = await executeDontEndSet500OnError(
+    `SELECT * FROM blogs ORDER BY id DESC`,
```

Comment:
Reduce the payload. This content will be very large. You can just fetch first 100 characters of the body using `SUBSTRING` method